global.json

```diff
@@ -1,24 +1,22 @@
 {
   "sdk": {
-    "version": "9.0.100-alpha.1.23557.5"
+    "version": "8.0.100-rtm.23506.1"
   },
   "tools": {
-    "dotnet": "9.0.100-alpha.1.23557.5",
+    "dotnet": "8.0.100-rtm.23506.1",
     "runtimes": {
       "dotnet/x64": [
-        "8.0.0-rtm.23516.15",
-        "$(MicrosoftNETCoreAppRuntimewinx64Version)"
+        "6.0.22"
       ],
       "aspnetcore/x64": [
-        "8.0.0-rtm.23517.6",
-        "$(MicrosoftAspNetCoreAppRuntimewinx64Version)"
+        "6.0.22"
       ]
     }
   },
   "msbuild-sdks": {
     "Microsoft.Build.NoTargets": "3.7.0",
     "Microsoft.Build.Traversal": "3.2.0",
-    "Microsoft.DotNet.Arcade.Sdk": "9.0.0-beta.23557.7",
-    "Microsoft.DotNet.Helix.Sdk": "9.0.0-beta.23557.7"
+    "Microsoft.DotNet.Arcade.Sdk": "8.0.0-beta.23525.4",
+    "Microsoft.DotNet.Helix.Sdk": "8.0.0-beta.23525.4"
```

Comment:
I'd keep the newer Arcade, if possible

Reply:
9.0 Arcade will start depending on the 9.0 SDK soon, and this branch should only consume 8.0 things, so I've made this branch subscripte t o 8.0 band SDK updates like other dotnet servicing branches.